karbon/karbon_cluster_registration_service_test.go

```diff
@@ -15,14 +16,15 @@ import (
 	"github.com/nutanix-cloud-native/prism-go-client/internal/testhelpers"
 )
 
+var (
+	NoCategoryMappingError = errors.New("error: map[Cluster Create:Failed to validate the category mapping. No category mappings provided.]")
+	NoUUIDError            = errors.New("error: uuid in body is required")
```

Comment:
Do we want these to be exported as part of the Karbon client API?

Reply:
we don't these to be exported since those has changed from errors to string data type based on change from assert.Equal to assert.Contains.